src/topology/uniform_space/cauchy.lean

```diff
@@ -39,6 +39,14 @@ lemma cauchy_iff {f : filter α} :
   cauchy f ↔ (ne_bot f ∧ (∀ s ∈ 𝓤 α, ∃t∈f, t ×ˢ t ⊆ s)) :=
 cauchy_iff'.trans $ by simp only [subset_def, prod.forall, mem_prod_eq, and_imp, id, ball_mem_comm]
 
+lemma cauchy.to_ultrafilter {l : filter α} (h : cauchy l) :
```

Comment:
I think this name is a bit misleading, it sounds like the type of `h.to_ultrafilter` will be `ultrafilter α`

Reply:
What about `cauchy.ultrafilter_of`? Or should I use `ultrafilter.cauchy_of`?